contracts/src/Zappers/Modules/Exchanges/UniV3Exchange.sol

```diff
@@ -52,9 +52,7 @@ contract UniV3Exchange is LeftoversSweep, IExchange, IUniswapV3SwapCallback {
         returns (uint256)
     {
         // See: https://github.com/Uniswap/v3-core/blob/d8b1c635c275d2a9450bd6a78f3fa2484fef73eb/contracts/UniswapV3Pool.sol#L608
-        //uint160 sqrtPriceLimitX96 = _zeroForOne(boldToken, collToken) ? MIN_SQRT_RATIO + 1: MAX_SQRT_RATIO - 1;
-        uint256 maxPrice = _maxBoldAmount * DECIMAL_PRECISION / _minCollAmount;
-        uint160 sqrtPriceLimitX96 = priceToSqrtPrice(boldToken, collToken, maxPrice);
+        uint160 sqrtPriceLimitX96 = _zeroForOne(boldToken, collToken) ? MIN_SQRT_RATIO + 1: MAX_SQRT_RATIO - 1;
```

Comment:
Can't we just pass 0?

https://github.com/Uniswap/v3-periphery/blob/0682387198a24c7cd63566a2c58398533860a5d1/contracts/lens/Quoter.sol#L134-L135

Reply:
Fixed.